cachito/web/models.py

```diff
@@ -963,9 +487,6 @@ def from_json(cls, kwargs):
         if not isinstance(dependency_replacements, list):
             raise ValidationError('"dependency_replacements" must be an array')
 
-        for dependency_replacement in dependency_replacements:
-            Dependency.validate_replacement_json(dependency_replacement)
```

Comment:
I would like to keep all the validation if possible. Since the `Dependency` class is being removed, could you just extract the `validate_replacement_json` method to a separate function?